README.md

```diff
@@ -103,17 +97,17 @@ dbt test
    - Create a directory `.dbt` in your root directory if one doesn't exist already, then create a `profiles.yml` file in `.dbt` 
    - Add the following block to the file:
 ```yaml
-  synthea_omop_etl:
+synthea_omop_etl:
   outputs:
     dev:
       dbname: <name of local postgres DB>
       host: <host, e.g. localhost>
-      pass: <password>
+      user: <username>
+      password: <password>
       port: 5432
       schema: dbt_synthea_dev
-      threads: 1
+      threads: 8
```

Comment:
Postgres can handle more than 1 connection.

Reply:
maybe we let users choose? and specify above they can choose appropriate # threads?

Reply:
Good idea! For postgres 1 is a slow default! Let me add a note